pkgs/development/python-modules/array-record/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, absl-py
+, etils
+, importlib-resources
+, pythonOlder
+, typing-extensions
+, zipp
+}:
+
+buildPythonPackage rec {
+  pname = "array-record";
+  version = "0.4.0";
+  format = "wheel";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit version format;
+    pname = "array_record";
+    dist = "py310";
+    python = "py310";
```

Comment:
This does not work for python 3.11 or pypy. Not sure if python has attrs for that.

Reply:
The binaries for python 3.11 are not distributed.
I have set `disabled = pythonVersion != "3.10".